protos/local/temporal/sdk/core/workflow_completion/workflow_completion.proto

```diff
@@ -20,10 +21,14 @@ message WorkflowActivationCompletion {
 message Success {
     // A list of commands to send back to the temporal server
     repeated workflow_commands.WorkflowCommand commands = 1;
+    // Any internal flags which the lang SDK used in the processing of this activation
+    repeated uint32 used_internal_flags = 6;
```

Comment:
I am probably just misreading or not finding a macro somewhere or something, but with a quick find-text I don't see where these are put on the WFT completion to the server.

Reply:
Here: https://github.com/temporalio/sdk-core/pull/482/files#diff-db8d7ea3d79aee3cbe4019c0292acc44a5f4e55de5aa225be9d2285b4cc62724R995

Reply:
Oh wait, you're right, inside that it's not gathering it up right. Good catch!